Helper/DeviceView.php

```diff
@@ -54,14 +54,12 @@ class DeviceView
      */
     public function __construct(Container $serviceContainer)
     {
-        if (false === $serviceContainer->isScopeActive('request')) {
+        if (!$this->request = $serviceContainer->get('request_stack')->getMasterRequest()) {
```

Comment:
A solution might be to register a request event listener which populates the current request to the `DeviceView` by calling a method like `setRequest()` (to be introduced).